assets/msalv2.js

```diff
@@ -173,11 +173,51 @@ var aadOauth = (function () {
       .catch(onError);
   }
 
-  function getAccessToken() {
+  async function getAccessToken() {
+    const account = getAccount();
+
+    if (account !== null && authResult === null) {
+      try {
+        // Silent acquisition only works if we the access token is either
+        // within its lifetime, or the refresh token can successfully be
+        // used to refresh it. This will throw if the access token can't
+        // be acquired.
+        const silentAuthResult = await myMSALObj.acquireTokenSilent({
+          scopes: tokenRequest.scopes,
+          prompt: "none",
+          account: account,
+          extraQueryParameters: tokenRequest.extraQueryParameters
+        });
+
+        authResult = silentAuthResult
+      } catch (_) {
+      }
+    }
+
     return authResult ? authResult.accessToken : null;
   }
 
-  function getIdToken() {
+  async function getIdToken() {
+    const account = getAccount();
+
+    if (account !== null && authResult === null) {
```

Comment:
Please factor this common code from getAccessToken and getIdToken out into a separate function

Reply:
Introduced silentlyAcquireToken() and referenced it from login(), getAccessToken() and getIdToken()

https://github.com/Earlybyte/aad_oauth/pull/215/commits/b5baf8080449f5786ab29f7f1aed952e8a9b6d19